apistructs/pipeline.go

```diff
@@ -364,9 +364,11 @@ type PipelinePageListRequest struct {
 	CountOnly bool `schema:"countOnly"` // 是否只获取 total
 
 	// internal use
-	SelectCols []string `schema:"-" ` // 需要赋值的字段列表，若不声明，则全赋值
-	AscCols    []string `schema:"ascCol"`
-	DescCols   []string `schema:"descCol"`
+	SelectCols   []string `schema:"-" ` // 需要赋值的字段列表，若不声明，则全赋值
+	AscCols      []string `schema:"ascCol"`
+	DescCols     []string `schema:"descCol"`
+	StartIDBegin uint64   `schema:"-"`
```

Comment:
unclear var name.
include or exclude?
refer to `gt, ge, lt, le`

Reply:
adjusted, use 'gt' 'lt'